spec/services/reporter_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Reporter do
+  TEST_START_TIME = DateTime.now # useful for both output cleanup and CSV filename testing
```

Comment:
How about making this: `let!(:start_time) { DateTime.now }`?  and changing `after(:all)` to `after(:each)`? Would that work?

Reply:
i'll give that a shot!  first thing i tried was `let(:start_time) { DateTime.now }`, and rspec complained about `let` usage in an `after`.  but it may have been specific to `after(:all)`, can't remember for sure.

Reply:
that seemed to work, though it also seemed to make consistently flappy a test that was probably prone to flapping anyway.  have a (hopefully acceptable) fix for that too, will push changes shortly...